lib/trufi_configuration.dart

```diff
@@ -107,3 +108,22 @@ class TrufiConfigurationUrl {
   String donate = "";
   String share = "";
 }
+
+class TrufiConfigurationDrawer {
+  DefinitionFeedBack definitionFeedBack;
+}
+
+enum FeedBackType { email, url }
+
+class DefinitionFeedBack {
+  /// You can selected the type feedback with enum
+  /// [type] parameter support {email and url} 
+  final FeedBackType type;
+  /// [body] is you url or email for feedback.
+  final String body;
+
+  DefinitionFeedBack({
```

Comment:
In order to enforce usage rather than just make it "required" you could just use the not named constructor.
```suggestion
   DefinitionFeedBack(this.type, this.body);
```

Reply:
I add the change in a new commit, because we need to remove the "import 'package:meta/meta.dart';"